server.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+import 'messaging-app';
```

Comment:
This isn't necessary - and will cause a syntax error when node runs the file. Also, you'll need to import the `connect` function you refer to below:

```suggestion
const connect = require("./db");
```